website/content/docs/other-specifications/quota.mdx

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+page_title: Nomad Quota Specification
+description: Learn about Nomad's Resource Quota specification.
+---
+
+# Nomad Quota Specification
+
+<EnterpriseAlert>
+Resource Quotas are only available with Nomad Enterprise.
+</EnterpriseAlert>
+
+[Resource Quotas](/nomad/docs/enterprise#resource-quotas)
+enable an operator to limit resource consumption across teams
+or projects to reduce waste and align budgets.
+
+Quotas can be managed using the Nomad CLI via the
+[`quota`](/nomad/docs/commands/quota)
+subcommands, or with Nomad's
+[HTTP API](/nomad/api-docs/quotas)
+directly.
+
+You may also follow a
+[Quotas tutorial](/nomad/tutorials/governance-and-policy/quotas)
+for more example usage.
+
+## Example Quota Specification
+
+The CLI command `nomad quota init` generates an example `spec.hcl` file:
+
+```hcl
+name        = "default-quota"
+description = "Limit the shared default namespace"
+
+# Create a limit for the global region. Additional limits may
+# be specified in-order to limit other regions.
+limit {
+  region = "global"
+  region_limit {
+    cores      = 0
+    cpu        = 2500
+    memory     = 1000
+    memory_max = 1000
+    device "nvidia/gpu/1080ti" {
+      count = 1
+    }
+  }
+  variables_limit = 1000
+}
+```
+
+## Quota Specification Parameters
+
+- `name` `(string)` - The name of the Quota. This will be used to connect it to a
+[`Namespace`](/nomad/docs/other-specifications/namespace).
+- `description` `(string: <optional>)` - A human-readable description.
+- `limit` <code>([QuotaLimit](#limit-parameters))</code> - The resource limit in a Nomad region.
+
+### `limit` Parameters
+
+- `region` `(string)` - The Nomad `region` that the limit applies to.
+- `region_limit` <code>([RegionLimit](#region_limit-parameters))</code> - Resources to limit.
+- `variables_limit` `(int: 0)` - Maximum total size of all Nomad
+[`variables`](/nomad/docs/concepts/variables)
+in MiB. The default `0` means unlimited, and `-1` means variables are fully disabled.
+
+### `region_limit` Parameters
+
+These are associated with the parameters on a task
+[`resources`](/nomad/docs/job-specification/resources)
+block in a job specification, and the
+[CPU concepts](/nomad/docs/concepts/cpu)
+page has more information regarding CPU resources as well.
+
+- `cores` `(int: <optional>)` - Number of CPU cores
+- `cpu` `(int: <optional>)` - Amount of CPU in MHz
+- `memory` `(int: <optional>)` - Amount of memory in MB
+- `memory_max` `(int: <optional>)` - Max amount of memory in MB
+- `device` <code>([Device](#device-parameters): nil)</code>
```

Comment:
I'd probably break the leading paragraph into 2 sentences. The "as well" doesn't seem to follow from the "are associated with". We don't really have good _reference_ docs on what the heck quotas do (just the tutorials), so this seems like a good place for it.

```suggestion
Region limits define the limits on cumulative task `resources` blocks for allocations 
in the namespace and region the quota is applied to.

- `cores` `(int: <optional>)` - The limit on total number of CPU cores from all 
  `resources.cores` in the namespace. The [CPU concepts](/nomad/docs/concepts/cpu) 
  documentation has more details on CPU resources.
- `cpu` `(int: <optional>)` - The limit on total amount of CPU from all `resources.cpu`
  in the namespace.
- `memory` `(int: <optional>)` - The limit on total mount of memory in MB from all
  `resources.memory` in the namespace.
- `memory_max` `(int: <optional>)` - The limit on total mount of hard memory limits in MB 
  from all `resources.memory_max` in the namespace.
- `device` <code>([Device](#device-parameters): nil)</code>
```

The `resources.memory_max` bit here feels like it could use more precision but I'm not super familiar with how that's implemented either.

Reply:
I was leaning on the `job-specification/resources` doc to elaborate on these details, but I like your more detailed info!

Reply:
8071c2a9590aeb869845270ea5f71880387e902c